src/db/types.rs

```diff
@@ -0,0 +1,43 @@
+//! Extra types defined in postgres
+//!
+//! db-sync defines a few types in postgres. When diesel generates a schema, it uses these types,
+//! but since they don't exist on the Rust side, type checking fails. So we provide them in this
+//! module
+//!
+//! Many types here have slightly wonky capitalization, but that's just what diesel expects
+
+use diesel_derive_enum::DbEnum;
+
+
+#[derive(Debug, Clone, Copy, DbEnum, SqlType)]
+pub enum Rewardtype {
```

Comment:
@cameron1024 Shouldn't we capitalize `type` in all types here`RewardType`, `ScriptpurposeType ` etc., for better readability?

Reply:
IMO this is a bug in the db-sync schema. db-sync has postgres types like `rewardtype`, but arguably they should be called `reward_type`. Diesel sees these postgres types and concludes that "rewardtype" is one word, so expects structs to be called `Rewardtype`. 

I considered exporting aliases like `pub type RewardType = Rewardtype`, but these types aren't actually used anywhere, they're only needed to make `schema.rs` compile, so it didn't seem worth it